signbank/dictionary/forms.py

```diff
@@ -95,7 +95,7 @@ class GlossSearchForm(forms.ModelForm):
     trans_lang = forms.ModelChoiceField(required=False, empty_label=_('Choose language'), queryset=Language.objects.all())
 
     # Adding topics
-    semantic_field = forms.ModelMultipleChoiceField(label=_('Topic'), queryset=FieldChoice.objects.filter(field='semantic_field'),
+    semantic_field = forms.ModelMultipleChoiceField(label=_('Topic'), queryset=FieldChoice.objects.filter(field='semantic_field').order_by('english_name'),
```

Comment:
@jonholdsworth do you need this `order_by` as well as the `Meta` class definition?

Reply:
Yes. The Meta ordering does not affect this page.